src/py/flwr/common/args.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2024 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Common Flower arguments."""
+
+import argparse
+import sys
+from logging import DEBUG, WARN
+from os.path import isfile
+from pathlib import Path
+from typing import Optional
+
+from flwr.common.logger import log
+
+
+def add_args_flwr_app_common(parser: argparse.ArgumentParser, flwr_app: str) -> None:
+    """Add common Flower arguments for flwr-*app to the provided parser."""
+    parser.add_argument(
+        "--run-once",
+        action="store_true",
+        help=f"When set, this process will start a single {flwr_app} "
+        "for a pending Run. If no pending run the process will exit. ",
```

Comment:
We should drop the last sentence. `flwr-serverapp` does wait until a pending run is pulled via `PullServerAppInputs`. My understanding is that an equivalent (but at the `Message` level instead at the `Run` level) happens in `flwr-clientapp` via the token.

Reply:
I've moved this back to the respective `flwr-*app` since the behaviour is slightly different. `flwr-serverapp` is `Run`-based, whereas `flwr-clientapp` is `token`-based.